### PR TITLE
Add telemetry for property compression

### PR DESCRIPTION
**Pull request by DavIvek**
Adds telemetry for property compression.

### CODE REVIEW

**Comment by DavIvek** [comment]
---
__*Leave above in PR description, copy the below into a comment*__
___


### Tracking
- [ ] **[Link to Epic/Issue]**


### Standard development
- [ ] Update unit/E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch this branch


### CI Testing Labels
- [ ] Select the appropriate CI test labels _(CI -build=**build-name** -test=**test-suite**)_


### Documentation checklist
- [ ] Add the documentation label
- [ ] Add the bug / feature label
- [ ] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [ ] Write a release note, including added/changed clauses
    - **[ Release note text ]**
- [ ] **[ Documentation PR link memgraph/documentation#XXXX ]**
    - [ ] Is back linked to this development PR
- [ ] **[ Tag someone from docs team ]**

File: src/dbms/dbms_handler.hpp [hunk 1]
```diff
@@ -85,6 +87,11 @@ static inline nlohmann::json ToJson(const Statistics &stats) {
                              {storage::IsolationLevelToString((storage::IsolationLevel)1), stats.isolation_levels[1]},
                              {storage::IsolationLevelToString((storage::IsolationLevel)2), stats.isolation_levels[2]}};
   res["durability"] = {{"snapshot_enabled", stats.snapshot_enabled}, {"WAL_enabled", stats.wal_enabled}};
+  res["property_store_compression_enabled"] = stats.property_store_compression_enabled;
+  res["property_store_compression_level"] = {
+      {utils::CompressionLevelToString((utils::CompressionLevel)0), stats.property_store_compression_level[0]},
```
Comment (Ignition) [inline src/dbms/dbms_handler.hpp:92]: Don't cast `(utils::CompressionLevel)0` just use `utils::CompressionLevel::LOW`

File: src/dbms/dbms_handler.hpp [hunk 2]
```diff
@@ -317,6 +324,8 @@ class DbmsHandler {
         ++stats.isolation_levels[(int)storage_info.isolation_level];
         stats.snapshot_enabled += storage_info.durability_snapshot_enabled;
         stats.wal_enabled += storage_info.durability_wal_enabled;
+        stats.property_store_compression_enabled += storage_info.property_store_compression_enabled;
+        ++stats.property_store_compression_level[(int)storage_info.property_store_compression_level];
```
Comment (Ignition) [inline src/dbms/dbms_handler.hpp:328]: Use UnderlyingCast

File: src/utils/compressor.cpp [hunk 3]
```diff
@@ -45,14 +45,40 @@ bool ValidStoragePropertyStoreCompressionLevel(std::string_view value) {
...
+      return Z_BEST_SPEED;
+    case CompressionLevel::MID:
+      return Z_DEFAULT_COMPRESSION;
+    case CompressionLevel::HIGH:
+      return Z_BEST_COMPRESSION;
+  }
+  return Z_DEFAULT_COMPRESSION;
+}
+
+std::string CompressionLevelToString(CompressionLevel level) {
```
Comment (Ignition) [inline src/utils/compressor.cpp:70]: use string_view

**Review by Ignition — APPROVED** [review]

**Pull request merged**